doc/api/http2.md

```diff
@@ -1873,6 +1873,15 @@ A value of `0` will disable the timeout behavior on incoming connections.
 The socket timeout logic is set up on connection, so changing this
 value only affects new connections to the server, not any existing connections.
 
+#### `server.updateSettings([settings])`
+* `settings` {HTTP/2 Settings Object}
```

Comment:
```suggestion
<!-- YAML
added: REPLACEME
-->

* `settings` {HTTP/2 Settings Object}
```
And the same for the other property. (the `REPLACEME` is used as is and will be replaced by tooling later)

Reply:
> And the same for the other property. (the `REPLACEME` is used as is and will be replaced by tooling later)

Ah! Thank you, I would have never known. I need to look deeper at the documentation for contributing

Reply:
No problem, it's easy to miss such things :+1:.

I think this is partly documented in https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#step-3-code
> If you are adding to or deprecating an API, use REPLACEME for the version number in the documentation YAML.

Though I don't think we mention *adding* the yaml description in the first place besides this
https://github.com/nodejs/node/blob/master/doc/guides/doc-style-guide.md
> When documenting APIs, update the YAML comment associated with the API as appropriate. This is especially true when introducing or deprecating an API.